README.md

```diff
@@ -25,7 +25,7 @@ https://github.com/user-attachments/assets/69f34f33-14e5-4bcd-a96e-8b069e1727ef
 
 ## Installation
 
-**Enable experimental API for the extension**. This is needed for displaying devcontainers in Remote Explorer.
+**Enable experimental API for the extension**. This is needed for displaying devcontainers in Remote Explorer. Create or modify `.vscode/argv.json` with the following content in your project:
```

Comment:
I'd suggest "Create or modify `argv.json` file with the following content: ". I suggest so because the path to the argv.json file depends on the editor the user uses. I, for instance, don't have the `.vscode` directory as I use VS Codium and have `vscode-oss` instead. To correctly identify the argv.json file one must use `Preferences: Configure Runtime Arguments` command, that's mentioned next to the snippet.